metatron/src/test/java/uk/gov/ida/eidas/metatron/apprule/rules/CreatesConfigDropwizardClientRule.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.ida.eidas.metatron.apprule.rules;
+
+import io.dropwizard.testing.junit.DropwizardClientRule;
+import org.junit.Assert;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URI;
+
+public class CreatesConfigDropwizardClientRule extends DropwizardClientRule {
+    public final static String CONFIG_LOCATION = "/tmp/countriesConfig.yaml";
```

Comment:
Might be worth using [createTempDirectory()](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute...)) - potentially more portable as it apparently uses the systems default temp dir.

Reply:
Good suggestion.